consensus/src/experimental/buffer_item.rs

```diff
@@ -154,15 +166,34 @@ impl BufferItem {
                         commit_proof,
                         callback,
                     }))
-                } else {
+                } else if validator
+                    .check_voting_power(unverified_signatures.signatures().keys())
+                    .is_ok()
+                {
+                    let commit_proof = aggregate_ledger_info(
+                        &generate_commit_proof(&commit_info, &ordered_proof),
```

Comment:
this name becomes confusing now, we probably should call it generate_commit_ledger_info